code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -80,6 +86,14 @@
 				return
 			accesses -= SSid_access.get_region_access_list(list(region))
 			. = TRUE
+		if("passedName")
+			var/new_name = trim(params["passedName"], 30)
+			passed_name = new_name
```

Comment:
this isn't sanitized. This needs to be sanitized

Reply:
Also checked the other threads from mothblocks, I'm talking specifically about html.
This lets you apply html that gets executed on clients if they examine the object, you need to html encode it.